tests/test_privileges.py

```diff
@@ -0,0 +1,58 @@
+from nose.tools import assert_false, assert_true, raises
+import datajoint as dj
+from os import environ
+from . import schema
+
+namespace = locals()
+
+
+class TestUnprivileged:
+
+    def __init__(self):
+        self.connection = None
+        self.previous_connection = None
+
+    def setUp(self):
+        # a connection with only SELECT privileges.  This user must be defined through mysql with only SELECT
+        # privileges to djtest schemas
+        view_connection = dict(
+            host=environ.get('DJ_TEST_HOST', 'localhost'),
+            user='djview',
+            password='djview')
+        self.previous_connection = dj.conn.connection if hasattr(dj.conn, 'connection') else None
+        self.connection = dj.conn(**view_connection, reset=True)
```

Comment:
Same as below - won't work in Python 3.4 or below. Change with `dj.conn(reset=True, **view_connection)`.